### PR TITLE
Update share-calendars-with-external-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added instructions to share calendar to external users

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8782

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 1]
```diff
@@ -52,7 +52,21 @@ Instant syncing for sharing outside your tenant isn't supported currently. While
 
 ## Invite people to access calendars
 
-Once sharing is enabled, calendar owners can extend invitations to specific users. For instructions, see [Sharing your calendar in Outlook Web App](https://support.microsoft.com/office/7ecef8ae-139c-40d9-bae2-a23977ee58d5).
+Once sharing is enabled, calendar owners can extend invitations to specific users.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
Once sharing is enabled, calendar owners can extend invitations to specific users.
```

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 2]
```diff
@@ -52,7 +52,21 @@ Instant syncing for sharing outside your tenant isn't supported currently. While
...
+- Choose how much information you want this person to see
+
+    1) Can view when I'm busy lets the person see when you are busy but does not include details such as the event location.
+
+    2) Can view titles and locations lets the person see when you are busy as well as the title and location of events.
+
+    3) Can view all details lets the person see all the details of your events.
+
+    4) Can edit lets the person see all details of your events and edit your calendar (only available when sharing with people in your organization).
+- Select **Share**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
1. Open [Outlook on the Web](https://outlook.office365.com).

2. At the top of the page, select the app launcher and select **Calendar**. By default, your primary calendar is called "Calendar". If you created other calendars, you can select one of them to share instead. You cannot share calendars owned by other people.

3. Enter the name or email address of the person with whom you want to share your calendar in the **Send a sharing invitation in email** box.

4. Choose how much information you want this person to see:

- **Can view when I'm busy** lets the person see when you are busy but does not include details such as the event location.

- **Can view titles and locations** lets the person see when you are busy as well as the title and location of events.

- **Can view all details** lets the person see all the details of your events.

- **Can edit** lets the person see all details of your events and edit your calendar (only available when sharing with people in your organization).

5. Select **Share**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 3]
```diff
@@ -52,7 +52,25 @@ Instant syncing for sharing outside your tenant isn't supported currently. While
...
+
+4. Choose how much information you want this person to see:
+
+- **Can view when I'm busy** lets the person see when you are busy but does not include details such as the event location.
+
+- **Can view titles and locations** lets the person see when you are busy as well as the title and location of events.
+
+- **Can view all details** lets the person see all the details of your events.
+
+- **Can edit** lets the person see all details of your events and edit your calendar (only available when sharing with people in your organization).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
     - **Can view when I'm busy** lets the person see when you are busy but does not include details such as the event location.

     - **Can view titles and locations** lets the person see when you are busy as well as the title and location of events.

     - **Can view all details** lets the person see all the details of your events.

     - **Can edit** lets the person see all details of your events and edit your calendar (only available when sharing with people in your organization).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**